### ISSUE TITLE
CC | tests: Add s390x support for authenticated registry tests

### ISSUE THREAD

**Issue by stevenhorsman** [kata-containers/tests#5291]
The new test case added in https://github.com/kata-containers/tests/issues/5209 only had test images for s390x and this feature isn't currently available in s390x until https://github.com/confidential-containers/image-rs/issues/80 is resolved, but after that we should update the test cases to work for x86 and s390x.

**Comment by stevenhorsman** [comment]
FYI @BbolroC - I've created this issue to track the work I mentioned in the other PR for this.

**Comment by stevenhorsman** [comment]
The script I used to generate the multi-arch auth image was:
```sh
#!/usr/bin/env bash

# Script to push multi-arch manifest image based on busybox to a repo
# Requires QUAY_CREDS set to <user>:<password> to push

set -o errexit
set -o pipefail
set -o nounset

registry="${registry:-quay.io/kata-containers/confidential-containers-auth}"

supported_arches=(
	"amd64"
	"s390x"
)

function push_multiarch_test_images() {
	tag=test #$(date +%Y%m%d%H%M%s)
	image_name=${registry}:${tag}
	buildah manifest rm ${image_name}
	buildah manifest create ${image_name}
	local arch_images=()

	for arch in ${supported_arches[@]}; do
		echo "Pushing authenticated test image for ${arch}"
		skopeo copy --insecure-policy "docker://docker.io/library/busybox:latest" --override-arch ${arch} docker://${registry}:${arch}-${tag} --dest-creds=${QUAY_CREDS}
		buildah manifest add ${image_name} docker://${registry}:${arch}-${tag} --creds ${QUAY_CREDS}
	done

	buildah manifest push --all ${image_name} docker://${image_name} --remove-signatures --creds ${QUAY_CREDS}
}

function main() {
	push_multiarch_test_images
}

main "$@"
```

**Comment by stevenhorsman** [comment]
Closed with #5292 merging
